notevenalex/book-recipe/sass-folder/styles.scss

```
@@ -0,0 +1,63 @@
+@import "../variables.scss";
+
+header{
+  h2 {background-color: "$red-1";
+    width: 40%;
+}
+}
+body {
+  position: relative;
+  background-color: "$beige-1";
```

```suggestion
  background-color: $beige-1;
```
No uses `""` para las variables